proto/README.md

```diff
@@ -1,11 +1,52 @@
-## Protos
+# Provenance Blockchain Protobuf Install and Build
 
-The protos that get built come from the third_party folder.
+The Provenance Explorer uses a combination of [Provenance](https://github.com/provenance-io/provenance) and
+[Cosmos](https://github.com/cosmos/cosmos-sdk) [protobuf](https://developers.google.com/protocol-buffers) definitions.
+Protocol buffers (protobuf) are Google's language-neutral, platform-neutral, 
+extensible mechanism for serializing structured data.  The Provenance
+[gRPC](https://grpc.io) and protobuf provide the RPC mechanism that Provenance 
+Explorer (and all middleware, really) uses to communicate with the Provenance blockchain.
 
-To populate the folder with the most up to date protos, run `./scripts/copy_proto_v2.sh {prov_version} {cosmos_version}`.
+## Download Provenance Blockchain Protos
 
-This does not need to be run very often, but only when you feel the protos should be updated.
-This script will pull in appropriate protos from the given release version. Please verify that the copied protos are 
-correct for your purposes.
+This `proto` module compiles the protobuf definitions (protos) from the `third_party` directory.
+The compiled protos are then used in the Provenance Explorer `service` module
+to communicate with the blockchain.
 
-To build the protos for use, run `./gradlew proto:build`.
+Before compiling the protos, they must be downloaded locally.  The `third_party`
+directory contains the last download of the protos.  To update the `third_party`
+directory run this `gradle` task *from the root project directory*:
+
+```bash
+./gradlew proto:downloadProtos
+```
+
+> The `downloadProtos` task will clean the `third_party` directory prior to
+> download.  Do not edit the protos in that directory.
+
+This `gradle` task will download the Provenance and Cosmos proto versions defined
+in the `./buildSrc/src/main/kotlin/io/provenance/Versions.kt` file:
+
+```kotlin
+    //external protos
+    const val Provenance = "v0.3.0"
```

Comment:
Would be nice if we could reference the defined versions here, so at least theres a place someone could quickly look for current versions... dunno if thats possible.

Reply:
Added links to Provenance and Cosmos releases in the help message for the plugin.